### DISCUSSION TITLE
search should gives result based on current user language. if its english then only show english pages,if it is dutch then only show dutch pages

### DISCUSSION THREAD

**Discussion by shweta952** [requarks/wiki#4553] [Error / Bug Report] [answered]
<img width="632" alt="wiki" src="https://user-images.githubusercontent.com/92074118/136324901-34538a37-3df8-4342-a09f-7425c1387cbd.png">

**Comment by broxen** [comment]
Seems like you might want it to prioritize the selected language but still show these other language results? Or perhaps make this a toggle in settings: "Show only preferred language in search results."

Just imagining use-cases where a multi-lingual site might actually want these results displayed side by side.

**Comment by NGPixel** [comment, marked as answer]
Submit a new feature request [here](https://requarks.canny.io/wiki) instead.
